core/src/test/java/ar/com/wolox/wolmo/core/util/KeyboardUtilsTest.kt

```diff
@@ -0,0 +1,92 @@
+package ar.com.wolox.wolmo.core.util
+
+import android.app.Activity
+import android.content.Context
+import android.os.IBinder
+import android.view.inputmethod.InputMethodManager
+import android.widget.EditText
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.*
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.Implementation
+import org.robolectric.annotation.Implements
+import org.robolectric.shadow.api.Shadow
+import org.robolectric.shadows.ShadowActivity
+
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE)
+class KeyboardUtilsTest {
+
+    private var contextMock: Context? = null
```

Comment:
Use `lateinit` and assert it's not null.

Reply:
ok :+1: